src/main/java/br/edu/ifpb/pweb2/projeto/simpleevents/controller/EventoController.java

```diff
@@ -200,6 +216,41 @@ public ModelAndView cancelApplication(@PathVariable("id") Long id, Authenticatio
         return mav;
     }
 
+    @GetMapping("/avaliar/{id}")
+    public ModelAndView avaliarEventoForm(@PathVariable("id") Long id, Authentication auth) {
+        Evento evento = dao.findById(id).get();
+        List<Candidato> selecionados = candidatoDAO.findByVaga_EventoAndAprovacao(evento, Status.APROVADO);
+        ModelAndView mav = new ModelAndView("eventos/avaliar");
+        Usuario currentUser = getLoggedUser(auth);
+        Avaliacao avaliacao = new Avaliacao();
+        mav.addObject("avaliacao", avaliacao);
+        mav.addObject("idEvento", id);
+        for (Candidato selecionado : selecionados) {
+            if (selecionado.getUsuario().getUser_id() == currentUser.getUser_id()) {
```

Comment:
Comparação de Long deve ser feita usando o .equals()
https://stackoverflow.com/questions/20541636/comparing-boxed-long-values-127-and-128

> Java caches boxed Integer instances from -128 to 127. Since you are using == to compare objects references instead of values, only cached objects will match. Either work with long unboxed primitive values or use .equals() to compare your Long objects.